docker/entrypoint.sh

```diff
@@ -20,13 +20,13 @@ echo "Current UID/GID: $NEW_UID/$NEW_GID"
 
 # Create UNIX group on the fly if it does not exist
 if ! grep -q $NEW_GID /etc/group; then
-    groupadd --gid $NEW_GID $USER_NAME
+    sudo groupadd --gid $NEW_GID $USER_NAME
```

Comment:
Why is `sudo` required? I remember the container inside runs as root while building it?

Reply:
Agreed, these sudos aren't necessary. I've pushed a commit that removes them.